### PR TITLE
add newrelic plugin agent

### PR DESCRIPTION
**Pull request by snopoke**
@TylerSheffels @benrudolph

### CODE REVIEW

File: ansible/roles/nginx/templates/default.j2 [hunk 1]
```diff
@@ -1 +1,7 @@
 # {{ ansible_managed }}
+
+location /nginx_stub_status {
```
Comment (TylerSheffels) [inline ansible/roles/nginx/templates/default.j2]: :+1:

Comment (snopoke) [inline ansible/roles/nginx/templates/default.j2]: Note I haven't actually enabled the nginx monitoring (not sure of the best pattern to use to get apache on one proxy machine and nginx on the other). Any ideas would be helpful.

**Comment by TylerSheffels** [comment]
@snopoke Don't know enough about the plugin agent, but this looks awesome to me.  Maybe test conf on staging?

**Comment by benrudolph** [comment]
:clap: :clap:

**Comment by czue** [comment]
just got a demo of this. :clap:

**Pull request merged**